set.mm

```diff
@@ -319161,50 +319161,133 @@ can be the countable union of rare closed subsets (where rare means
-      ( cnvc wcel csca cfv ccms 3adant2 adantr cds cxp syl wceq eqid w3a cca wa
-      cmopn clm cdm wss cbn lssnvc cmt cres cngp csubg cnlm nvcnlm nlmngp clmod
-      cbs nvclmod lsssubg sylan subgngp syl2an2r ngpms 3ad2ant3 subgbas sqxpeqd
-      cms ressds reseq12d syl5eq eqcomd cme ngpmet eqeltrd simpr sylanbrc iscms
-      iscmet2 simpl3 resssca simpl2 eqeltrrd isbn syl3anbrc ) DIJZDKLZMJZCBJZUA
-      ZAUBLAUDLZUELUFUGZUCZEIJZEMJZEKLZMJEUHJWJWNWLWFWIWNWHBCDEFGUINOWMEUJJZEPL
-      ZEURLZWSQZUKZWSVHLZJWOWMEULJZWQWJXCWLWFWIXCWHWFDULJZWICDUMLJZXCWFDUNJXDDU
-      ODUPRWFDUQJWIXEDUSBCDGUTVAZCDEFVBVCNZOEVDRWMXAAXBWJXAASWLWJAXAWJADPLZCCQZ
-      UKXAHWJXHWRXIWTWIWFXHWRSWHCXHDEBFXHTVIVEWJCWSWJXECWSSWFWIXEWHXFNCDEFVFRVG
-      VJVKZVLOWMAWSVMLZJZWLAXBJWJXLWLWJAXAXKXJWJXCXAXKJXGXAEWSWSTZXATZVNRVOOWJW
-      LVPAWKWSWKTVSVQVOXAEWSXMXNVRVQWMWGWPMWMWIWGWPSWFWHWIWLVTCWGDEBFWGTWARWFWH
-      WIWLWBWCWPEWPTWDWE $.
-
-    $( Hilbert space property of a closed subspace: a closed subspace of a
+      ( wcel cfv ccms cds cxp cres cngp syl 3adant2 adantr wceq eqid cnvc cmopn
+      csca w3a cca clm cdm wss wa cbn simpl1 simpl2 cmt cbs csubg nvcnlm nlmngp
+      cnlm clmod nvclmod lsssubg sylan subgngp syl2an2r ressds 3ad2ant3 subgbas
+      cms ngpms sqxpeqd reseq12d syl5eq eqcomd cme ngpmet eqeltrd simpr iscmet2
+      sylanbrc iscms simpl3 cmslssbn syl22anc ) DUAIZDUCJKIZCBIZUDZAUEJAUBJZUFJ
+      UGUHZUIZWDWEEKIZWFEUJIWDWEWFWIUKWDWEWFWIULWJEUMIZELJZEUNJZWNMZNZWNVHJZIWK
+      WJEOIZWLWGWRWIWDWFWRWEWDDOIZWFCDUOJIZWRWDDURIWSDUPDUQPWDDUSIWFWTDUTBCDGVA
+      VBZCDEFVCVDQZREVIPWJWPAWQWGWPASWIWGAWPWGADLJZCCMZNWPHWGXCWMXDWOWFWDXCWMSW
+      ECXCDEBFXCTVEVFWGCWNWGWTCWNSWDWFWTWEXAQCDEFVGPVJVKVLZVMRWJAWNVNJZIZWIAWQI
+      WGXGWIWGAWPXFXEWGWRWPXFIXBWPEWNWNTZWPTZVOPVPRWGWIVQAWHWNWHTVRVSVPWPEWNXHX
+      IVTVSWDWEWFWIWABCDEFGWBWC $.
+
+    csschl.c $e |- ( Scalar ` W ) = CCfld $.
+    $( Hilbert space property of a complete subspace: a complete subspace of a
        complex pre-Hilbert space is a complex Hilbert space.  Remarks:  (a) In
-       contrast to ` ClSubSp ` , a closed (compact?) subspace is defined by "a
-       linear subspace in which all Cauchy sequences converge to a point in the
+       contrast to ` ClSubSp ` , a complete subspace is defined by "a linear
+       subspace in which all Cauchy sequences converge to a point in the
        subspace".  (b) This theorem does not hold for arbitrary subcomplex
        (pre-)Hilbert spaces, because the scalar field as restriction of the
        field of the complex numbers need not be closed.  (Contributed by NM,
        10-Apr-2008.)  (Revised by AV, 6-Oct-2022.) $)
-    csschl $p |- ( ( ( W e. CPreHil /\ ( Scalar ` W ) = CCfld /\ U e. S )
-                      /\ ( Cau ` D ) C_ dom ( ~~>t ` ( MetOpen ` D ) ) )
+    csschl $p |- ( ( W e. CPreHil /\ U e. S
+                     /\ ( Cau ` D ) C_ dom ( ~~>t ` ( MetOpen ` D ) ) )
                    -> ( X e. CHil /\ ( Scalar ` X ) = CCfld ) ) $=
```

Comment:
Superseded by cmslsschl

Reply:
Since this is the translation of the deprecated theorem of part 19, I would like to keep it (at least until the issue with the definition of closed subspaces is solved).

Reply:
idem: maybe put something in the comment

Reply:
> Superseded by cmslsschl

Not exactly! ~cmslsschl requires `W e. CHil` , whereas for ~csschl `W e. CPreHil` is sufficient.